server/matchmaker/matchmaker_queue.py

```diff
@@ -19,8 +19,9 @@
 
 
 class MatchmakerSearchTimer:
-    def __init__(self, queue_name):
-        self.queue_name = queue_name
+    def __init__(self, queue_name: str, players_in_queue: int):
```

Comment:
I considered changing the constructor to store a reference to the queue itself rather than store the number of players when the search starts, but I didn't want to hold open any potential connections.